src/components/DisplayColours.jsx

```diff
@@ -1,17 +1,19 @@
 import React from 'react'
+import TextSection from './TextSection'
 
 function DisplayColours({ colors }) {
   return (
     <>
+    <TextSection />
     <h3>Thank you for choosing Color Compass</h3>
     {colors.map(color => (
         <div key={color.hex}>
```

Comment:
This component may below inside of this map, passing in the `color` that the map is on at the moment.